psi4/src/export_mints.cc

```diff
@@ -644,6 +646,12 @@ void export_mints(py::module& m) {
           "Returns the multiplication of three matrics A, B, and C, with options to transpose each beforehand", "A"_a,
           "B"_a, "C"_a, "transA"_a = false, "transB"_a = false, "transC"_a = false);
 
+    py::enum_<DerivCalcType>(m, "DerivCalcType")
+        .value("Default", DerivCalcType::Default)
+        .value("SCF", DerivCalcType::SCF)
+        .value("SCFandDF", DerivCalcType::SCFandDF)
+        .value("Correlated", DerivCalcType::Correlated);
```

Comment:
fyi, starting fairly recently, one can finally add docstrings to enums in pb11. https://github.com/loriab/pylibefp/blob/master/pylibefp/core.cc#L493

Reply:
Done.